Material.Blazor/Components/DatePicker/InternalDatePickerPanel.razor

```diff
@@ -49,7 +48,6 @@
             {
                 <InternalDatePickerYearButton CurrentYear="@StartOfDisplayMonth.Year"
                                               CurrentYearId="@currentYearId"
-                                              DateIsSelectable="@DateIsSelectable"
```

Comment:
This was a copy-paste error: DateIsSelectable isn't even used in InternalDatePickerYearButton